assets/queries/dockerfile/missing_dnf_clean_all/query.rego

```diff
@@ -6,7 +6,7 @@ CxPolicy[result] {
 	command := resource.Value[0]
 
 	containsInstallCommand(command)
-
+	not containsDnfClean(input.document[i].command[name], resource.StartLine)
 	not containsCleanAfterInstall(command)
```

Comment:
From what I gather you intend to check if clean is not after the install whether the clean command is in the same RUN command or after, in another command. But seems to me that in the first check the query checks if the clean command is neither in the same command or after (which is correct), and in the second only checks if its not after the command. Is the second check necessary?